examples/arithmetic/tests/test_generated_bindings.rs

```diff
@@ -1,5 +1,5 @@
 uniffi_macros::build_foreign_language_testcases!(
-    ["src/arithmetic.udl",],
+    "uniffi-example-arithmetic",
```

Comment:
The vast majority of these tests now duplicate the crate name the tests are in - how do you feel about `build_foreign_language_testcases` just taking the list of test files and using `$CARGO_PKG_NAME` as the crate name, and if necessary, a slightly different name for the macro taking an explicit crate name if there are any that are not default?

Reply:
That's a great suggestion and it makes all the code much simpler.